debug.log

```diff
@@ -0,0 +1,15 @@
+[1126/182724.452:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
```

Comment:
I don't think this file should be in the remote. It should probably be part of gitignore.